tests/tiglFuselageSegment.cpp

```diff
@@ -712,3 +712,18 @@ TEST_F(TiglFuselageSegmentSimple, getSectionCenter)
     ASSERT_EQ(TIGL_MATH_ERROR, tiglFuselageGetSectionCenter(tiglHandle, "segmentD150_Fuselage_1Segment2ID", eta_out_of_range, &pointX, &pointY, &pointZ));
 }
 
+TEST_F(TiglFuselageSegmentSimple, getCenterLineLength)
+{
+    double centerLineLength = 0;
+
+    // test value of the result
+    ASSERT_NE(TIGL_NULL_POINTER, tiglFuselageGetCenterLineLength(tiglHandle, "SimpleFuselage", &centerLineLength));
```

Comment:
The first line cannot happen, if the second assert ist valid. Remove first line.